packages/spindle-ui/src/HeroCarousel/HeroCarouselItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback, FC } from 'react';
+import PropTypes from 'prop-types';
+import isEqual from 'lodash/isEqual';
+import './HeroCarouselItem.css';
+
+export type CarouselItem = {
+  imageUrl: string;
+  link: string;
+  title: string;
+};
+
+type Props = {
+  carouselItem: CarouselItem;
+  isLinkClicked: boolean;
+  itemLinkClassName: string;
+};
+
+export const HeroCarouselItem: FC<Props> = React.memo(
+  function HeroCarouselItem({
+    carouselItem,
+    isLinkClicked,
+    itemLinkClassName,
+  }) {
+    const handleLinkClick = useCallback(
+      (e: React.KeyboardEvent | React.MouseEvent) => {
+        if (!isLinkClicked) {
+          e.preventDefault();
+        }
+      },
+      [isLinkClicked],
+    );
+
+    return (
+      <li className="spui-HeroCarouselItem__ListItem">
+        <a
+          className={`${itemLinkClassName} spui-HeroCarouselItem__Link`}
+          href={carouselItem.link}
+          onClick={handleLinkClick}
+        >
+          <span className="spui-HeroCarouselItem__ImageBlock">
+            <img
+              alt=""
+              className="spui-HeroCarouselItem__Image"
+              src={`${carouselItem.imageUrl}?size=640`}
```

Comment:
```suggestion
              src={`${carouselItem.imageUrl}`}
```
[nits] ここのクエリパラメータはCDN依存の値のため汎用コンポーネントとしては削除した方が良さそうです！

Reply:
消しました！ -> https://github.com/openameba/spindle/pull/348/commits/a76648feebc91efdeb86d4d2fa41097418e60e47